### PR TITLE
clangxx: fix typo

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

fix a small typo in `clangxx`'s name

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.08.14" %}
+{% set version = "2018.08.21" %}
```
Comment (mbargull) [inline recipe/meta.yaml]: Do you think a build number bump is more appropriate?

Comment (CJ-Wright) [reply recipe/meta.yaml]: Not certain @conda-forge/core ?

Comment (scopatz) [reply recipe/meta.yaml]: New version is correct

**Comment by jakirkham** [comment]
Should we rebuild the webservice?

**Pull request merged**